src/components/ParticipantList/ParticipantList.tsx

```diff
@@ -169,30 +176,45 @@ export const ParticipantList = ({
           <div className={styler('divider')}></div>
 
           <div className={styler('formContainer')}>
-            <label htmlFor="role-change-select-menu">
-              <Text variant="heading" size="sm">
-                Role:
-              </Text>
-            </label>
-            <div className={styler('selectContainer')}>
-              <select
-                id="role-change-select-menu"
-                value={selectedRole}
-                onChange={handleInputChange}
-                className={styler('select')}
-                disabled={
-                  !localPeerRole || !localPeerRole.permissions.changeRole
-                }
-              >
-                <option value="" className="p-4">
-                  Select a new role
-                </option>
-                {roleNames.map(roleName => (
-                  <option value={roleName} key={roleName}>
-                    {roleName}
-                  </option>
-                ))}
-              </select>
+            <div className={styler('formRow')}>
+              <label htmlFor="role-change-select-menu">
+                <Text variant="heading" size="sm">
+                  Change role to:
+                </Text>
+              </label>
+              <div className={styler('selectContainer')}>
+                <select
+                  id="role-change-select-menu"
+                  value={selectedRole}
+                  onChange={handleInputChange}
+                  className={styler('select')}
+                  disabled={
+                    !localPeerRole || !localPeerRole.permissions.changeRole
```

Comment:
!localPeerRole?.permissions?.changeRole. Do we have to render this if permission not allowed?

Reply:
done